src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 7;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", 10);
+    }
+
+    public Figure getRandomFigure(int value) {
+
+        switch (value) {
+            case 1:
```

Comment:
Start case with 0, not 1
```suggestion
            case 0:
```